skops/card/_model_card.py

```diff
@@ -41,7 +41,7 @@ def _clean_table(table: str) -> str:
     # replace line breaks "\n" with html tag <br />, however, leave end-of-line
     # line breaks (eol_lb) intact
     eol_lb = "|\n"
-    placeholder = "$%!?"  # arbitrary string that never appears naturally
+    placeholder = "\x1f"  # unit separator control character
```

Comment:
I can't find this character, can you add something which would helps us find it in the tables, or add a description of what it is?

Reply:
Sure, in the Python ASCII control character library it appears [here](https://github.com/python/cpython/blob/3.11/Lib/curses/ascii.py#L36), I could link to this part of the source code, or even use the `curses.ascii` character in place if that would help readibility?

Reply:
Incredibly annoyingly, curses, the cpython module that supports ascii characters, [is not supported on windows](https://docs.python.org/3/howto/curses.html).

I've just added a small comment that refers to it in the curses library, and left it as a coded string instead of trying to use `ascii.US`